x/axelarnet/types/testutils/types.go

```diff
@@ -56,7 +58,7 @@ func RandomIBCDenom() string {
 // RandomIBCPath creates an IBC path
 func RandomIBCPath() string {
 	port := ibctransfertypes.PortID
-	identifier := fmt.Sprintf("%s%d", "channel-", rand.I64Between(0, 9999))
+	identifier := fmt.Sprintf("%s%d", "channel-", rand.PosI64())
```

Comment:
the ibc path is assembled manually in multiple places. extract it into a utils function?